cypress/e2e/shared/flows.test.ts

```diff
@@ -528,17 +529,32 @@ describe('Flows', () => {
       {
         panel: 'notification',
         menuItems: [...defaultMenuItems],
+        intercept: '@fetchSecrets',
       },
       {
         panel: 'schedule',
         menuItems: [...defaultMenuItems],
       },
     ]
+
+    // Intercepts
+    cy.intercept('/api/v2/buckets?*').as('fetchAllBuckets')
+    cy.intercept('/api/v2/orgs/*/secrets').as('fetchSecrets')
+
     items.forEach(item => {
       cy.getByTestID('panel-add-btn--1').click()
       // ugh.. cypress being cypress with ``
       const panelTestId = 'add-flow-btn--' + item.panel
-      cy.getByTestID(panelTestId).click()
+      if (item.panel === 'queryBuilder') {
+        cy.getByTestID(panelTestId).click()
+        cy.wait('@fetchAllBuckets')
+      } else if (item.panel === 'notification') {
+        cy.getByTestID(panelTestId).click()
+        cy.wait('@fetchSecrets')
+      } else {
+        cy.getByTestID(panelTestId).click()
+      }
```

Comment:
we can probably simplify this with two ifs for the waits with the click before it right?

cy.click()
if (query) wait
if (notification) wait